a11y-dialog.js

```diff
@@ -399,4 +399,4 @@
   } else if (typeof global === 'object') {
     global.A11yDialog = A11yDialog;
   }
-}(this));
+}(typeof global !== "undefined" ? global : window));
```

Comment:
Actually that’s not okay from me to ask that. I’m going to implement this fix right now while I have time so you can use it when you need. ;)